content/tooling/wrangler/secrets.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Secrets
+weight: 4
+new: true
+---
+
+### `put`
+
+Interactive command to create or replace a secret
```

Comment:
I'd add a line saying you can optionally pass an environment in. After all, the secrets API will also work without an environment variable attached. To be even more thorough, I would even potentially provide examples both with and without environment use below.